prow/cmd/phony/examples/review_approve_submitted.json

```diff
@@ -0,0 +1,320 @@
+{
+  "sender": {
```

Comment:
any chance we can remove the irrelevant fields so this is easier for a human to consume? It causes this otherwise pretty straight-forward PR to look enormous

Reply:
I'm certainly sympathetic to that concern, but I've recently been burned by the differences between my hand-rolled examples and real webhooks. :smiley: 

@fejta which fields are known to be irrelevant? I'll remove those. Alternately, we could remove these files and add to the `phony` docs instructions for retrieving webhook content.

Reply:
Another option: mark these files as generated so they're collapsed by default in PRs.

Reply:
Fine with any of these options.

Reply:
I added a `.gitattributes` to the repo root that marks these files as generated. You can see in the file view that they're now collapsed.